scripts/get_stress_test_summary.py

```diff
@@ -0,0 +1,46 @@
+import os
+import statistics
+from glob import glob
+from scripts.get_stats import get_stats, BenchmarkStats
+
+
+def get_results(folders):
+    results = BenchmarkStats()
+
+    for folder in folders:
+        stats = get_stats(folder)
+        results.get.extend(stats.get)
+        results.post.extend(stats.post)
+        results.total_requests = results.total_requests + stats.total_requests
+        results.total_failures = results.total_failures + stats.total_failures
+
+    results.average_get = statistics.mean(results.get)
+    results.average_post = statistics.mean(results.post)
+    results.average_total = statistics.mean(results.get + results.post)
+    results.error_percentage = (results.total_failures * 100) / results.total_requests
+
+    return results
+
+
+if __name__ == "__main__":
+    folder = os.getenv("OUTPUT_DIR")
+
+    if not folder:
+        print(
+            "'OUTPUT_DIR' environment variable must be provided e.g. outputs/daily-test"
+        )
+        sys.exit(1)
+
+    sorted_folders = sorted(glob(f"{folder}/*"))
+    result = get_results(sorted_folders)
+
+    summary = (
+        f'Questionnaire GETs average: {int(result.average_get)}ms\n'
+        f'Questionnaire POSTs average: {int(result.average_post)}ms\n'
+        f'All requests average: {int(result.average_total)}ms\n'
+        f'Total Requests: {int(result.total_requests)}\n'
+        f'Total Failures: {int(result.total_failures)}\n'
+        f'Error Percentage: {(round(result.error_percentage, 2))}%\n'
+    )
+
+    print(summary)
```

Comment:
We should remove this and add a dunder method to `BenchmarkStats` since this is a representation of the object to a user.
```py
    def __str__(self):
        return (
            f'Questionnaire GETs average: {int(self.average_get)}ms\n'
            f'Questionnaire POSTs average: {int(self.average_post)}ms\n'
            f'All requests average: {int(self.average_total)}ms\n'
            f'Total Requests: {int(self.total_requests)}\n'
            f'Total Failures: {int(self.total_failures)}\n'
            f'Error Percentage: {(round(self.error_percentage, 2))}%\n'
        )
```
then you simply call `print(result)`

```suggestion
    results = get_results(sorted_folders)
    print(results)
```